src/tests/webhook_test.ts

```diff
@@ -126,20 +127,38 @@ describe("Webhook", () => {
     });
   });
 
+  describe(".getSignatureHeader", () => {
```

Comment:
```suggestion
  describe("#getSignatureHeader", () => {
```

please use `#` for instance methods and `.` for static methods